src/main/resources/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# docker compose yaml file
+version: '3'
+services:
+
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: root
+      POSTGRES_USER: postgres
+      POSTGRES_DB: megaAppDb
+    networks:
+      - network-air
+
+  app:
+    image: openjdk:11
+    volumes:
+      - "C:/JetBrains/Java/Projects/MegaApplication/target:/home/"
```

Comment:
Could you move this `docker-compose.yml` to the very top of the project and use relative path to the `target`, please.

Reply:
...and this file could be removed)